communications/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+
+class EmailLog(models.Model):
+    email_type = models.CharField(max_length=255)
+    subject = models.CharField(max_length=255)
+    body = models.CharField(max_length=255)
+
+    timestamp = models.DateTimeField(auto_now_add=True)
+
+    recipient = models.ForeignKey(
+        "users.Registrations", 
+        related_name="email_log", 
```

Comment:
```suggestion
        related_name="email_logs", 
```

usually reverse names are plural `registration.email_logs`